src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue.cs

```diff
@@ -0,0 +1,168 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
```

Comment:
seems like that test should only be in the recoverability folder and not within retries?